src/test/groovy/ExampleSpec.groovy

```diff
@@ -20,10 +27,78 @@ class ExampleSpec extends BaseSpec {
             navigateToPracticeFormPage()
             PracticeFormPage practiceFormPage = at PracticeFormPage
         when: "Fill out the form"
-
-
             sleep(30)
         then: "Verify something"
             sleep(30)
     }
+
+    void "selectRandomColorFromDropdownMenu"(){
+        given: "User is on the Select Menu Page"
+            navigateToSelectMenuPage()
+        SelectMenuPage selectMenuPage = at SelectMenuPage
+        when:
+            selectMenuPage.chooseRandomNumber()
+        then:
+            sleep(3000)
```

Comment:
there should be some assertions in the then clause not sleep(3000). Update all the tests below as well. Think of meaningful assertions for your tests so that they verify that the tasks from the excercise are done.